library/src/main/java/com/owncloud/android/lib/resources/files/SearchRemoteOperation.java

```diff
@@ -54,7 +54,6 @@
         FAVORITE_SEARCH, // get all favorite files/folder
         RECENTLY_MODIFIED_SEARCH, // get files/folders that were modified within last 7 days, ordered descending by time
         PHOTO_SEARCH, // gets all files with mimetype "image/%"
-        SHARED_SEARCH, // show all shares
```

Comment:
Is this really not needed anylonger?
Will lead to crash here: https://github.com/nextcloud/android/blob/23a72722d340060c797881d56281baf74a4c26c2/app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java#L1701

Or is there a corresponding PR to remove it? :)

Reply:
Removed in https://github.com/nextcloud/android/pull/10015/files#diff-814a2c94173812cefd41e8a282e724941bbee46ae5c3332602936360609fe578L1701

It's only ever used in that `if`, which means it doesn't really do anything.

Reply:
I was sure that you already did this :) 
We then just need to make sure to merge #10015 before this one.

Reply:
> I was sure that you already did this :) We then just need to make sure to merge #10015 before this one.

That is a bit of a problem because #10015 requires the new attributes in this PR lol. I can remove this change and we'll clean it up at a later date.

Reply:
We have this from time to time. Then we usually just need to merge it within a very short time gap.
As it is not a mandatory change, it might be easier for this time to remove it.

Reply:
Reverted in https://github.com/nextcloud/android-library/pull/844/commits/e184d27168a58b9dbdb3dc6226ea5c4fb44e1acb